src/encoded/tests/test_schema_formats.py

```diff
@@ -0,0 +1,34 @@
+import glob
```

Comment:
This whole file is new. There were no tests of the `schema_formats.py` file that I could find. It makes me feel better to have some cross-check that the accession pairs are the right ones.